industrial_ci/src/util.sh

```diff
@@ -305,8 +305,12 @@ function ici_parse_jobs {
       _ici_parse_jobs_res="$3";;
   "true")
       _ici_parse_jobs_res="0";;
+  "True")
```

Comment:
We use `true` and `false` everywhere else.
Instead of accepting the Python syntax, it should be converted in the calling script.